template.tpl

```diff
@@ -619,6 +763,167 @@ ___TEMPLATE_PARAMETERS___
         "checkboxText": "EU Data Residency",
         "type": "CHECKBOX"
       },
+      {
+        "help": "Check this box to enable default event tracking. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-default-events\"\u003eRead more\u003c/a\u003e.",
+        "defaultValue": true,
+        "simpleValueType": true,
+        "name": "defaultEventTracking",
+        "checkboxText": "Enable default event tracking",
```

Comment:
I'm leaning on phrasing it like:

```suggestion
        "checkboxText": "Track Amplitude default events",
```

And same for all other DET like

* Track page views
* Track marketing attribution
...